netplugin/plugin/netplugin.go

```diff
@@ -339,15 +339,7 @@ func InitGlobalSettings(stateDriver core.StateDriver, inst *core.InstanceInfo) e
 		logrus.Warnf("Sleep 1 second and retry pulling global settings")
 		time.Sleep(1 * time.Second)
 	}
-	if inst.FwdMode == "" {
-		inst.FwdMode = gCfg.FwdMode
-	} else {
-		if inst.FwdMode != gCfg.FwdMode {
-			err := fmt.Errorf("Global forwarding mode setting doesn't match local settings, global=%v , local=%v", gCfg.FwdMode, inst.FwdMode)
-			logrus.Errorf(err.Error())
-			return err
-		}
-	}
+	inst.FwdMode = gCfg.FwdMode
```

Comment:
since it looks like we are going all in with config from etcd, should we warn if inst.FwdMode is configured whever that is created?

Reply:
there is no chance it would be configured though ... this function only got called when the netplugin starts, and we are not providing a way to set the fwdmode today